styles/widgets/generic/splitter.generic.less

```diff
@@ -0,0 +1,12 @@
+.dx-splitter {
+    width: 4px;
+    background-color: @datagrid-column-separator-bg;
```

Comment:
Do not use constants from other widgets, because their values can be changed by the customer in the process of creating new themes using a theme builder tool.